### ISSUE TITLE
Indent rule with object literals as parameters

### ISSUE THREAD

**Issue by zxlin** [eslint/eslint#8938]
**Tell us about your environment**

* **ESLint Version: 3.19.0**
* **Node Version: v6.11.0**
* **npm Version: 3.10.10**

**What parser (default, Babel-ESLint, etc.) are you using?**

default

**Please show your full configuration:**

<details>
<summary>Configuration</summary>

```js
{
  extends : 'airbnb',
  installedESLint : true,
  env : {
    node : true,
    es6 : true,
    mocha : true,
  },
}


```

</details>

**What did you do? Please include the actual source code causing the issue.**

```js
const User = new Model();

// Example 1 - Error
User
  .update(
    { _id: '123' },
    {
      $set : {                     // error  Expected indentation of 2 spaces but found 4  indent
        name : 'hello',            // error  Expected indentation of 4 spaces but found 6  indent
        email : 'test@test.com',
      },
    }                              // error  Expected indentation of 2 spaces but found 4  indent
  )
  .exec()
;

// Example 2 - Error
User
  .update(
    { _id: '123' },
    { $set : {                      // error  Expected indentation of 2 spaces but found 4  indent
      name : 'hello',
      email : 'test@test.com',
    } }
  )
  .exec()
;

// Example 3 - Good
User.update(
  { _id: '123' },
  { $set : {
    name : 'hello',
    email : 'test@test.com',
  } }
)
.exec();
```

**What did you expect to happen?**

No errors with all three examples

**What actually happened? Please include the actual, raw output from ESLint.**

```
   7:5  error  Expected indentation of 2 spaces but found 4  indent
   8:7  error  Expected indentation of 4 spaces but found 6  indent
  12:5  error  Expected indentation of 2 spaces but found 4  indent
  21:5  error  Expected indentation of 2 spaces but found 4  indent
```

**Comment by not-an-aardvark** [comment]
Thanks for reporting. This is a known issue and was fixed in ESLint 4.
